CMakeLists.txt

```diff
@@ -114,6 +136,21 @@ function(trusted_firmware_build)
     BUILD_BYPRODUCTS ${BUILD_BYPRODUCTS}
   )
 
+  set_target_properties(tfm PROPERTIES
+    BL2_BIN_PATH ${BL2_BIN_PATH}
+    BL2_HEX_PATH ${BL2_HEX_PATH}
+    BL2_ELF_PATH ${BL2_ELF_PATH}
+    TFM_S_BIN_PATH ${TFM_S_BIN_PATH}
+    TFM_S_HEX_PATH ${TFM_S_HEX_PATH}
+    TFM_S_ELF_PATH ${TFM_S_ELF_PATH}
+    TFM_NS_BIN_PATH ${TFM_NS_BIN_PATH}
+    TFM_NS_HEX_PATH ${TFM_NS_HEX_PATH}
+    TFM_NS_ELF_PATH ${TFM_NS_ELF_PATH}
+    TFM_S_SIGNED_BIN_PATH ${TFM_S_SIGNED_BIN_PATH}
+    TFM_NS_SIGNED_BIN_PATH ${TFM_NS_SIGNED_BIN_PATH}
+    TFM_S_NS_SIGNED_BIN_PATH ${TFM_S_NS_SIGNED_BIN_PATH}
+    )
```

Comment:
Please use: `_FILE` instead of `_PATH`.

Reply:
I agree `_FILE` is less ambiguous.

Reply:
> But we should document those properties / target files somewhere.

Good idea. Just comments, or is there a Cmake way I should do it?

Reply:
Done